### ISSUE TITLE
loss increased when tp > 1 for qwen1_5 continue pretrain 

### ISSUE THREAD

**Issue by changingivan** [alibaba/Pai-Megatron-Patch#217]
I compared Pai and HF-based framework for qwen1.5 model continue pretrain. I found that when tp > 1, the first step loss (before backward) increased. The model is qwen1.5-0.5B-Chat,  both seq_len and pad_len are set to be 8192, the length of training data are greater or equal to 8192,  batch_size (both micro and global) is 1.   Here's detail 

| Pai-Loss | HF-Loss | TP | PP |
|:--------:|:-------:|:--:|:--:|
|   3.07   |  2.92   | 1  | 1  |
|   3.12   |  2.92   | 2  | 1  |
|   3.35   |  2.92   | 4  | 1  |
|   4.46   |  2.92   | 8  | 1  |
|   3.10   |  2.92   | 1  | 2  |
|   3.09   |  2.92   | 1  | 4  |
|   3.13   |  2.92   | 1  | 8  |    

it's a tensor_parallel bug ？

**Comment by jerryli1981** [comment]
您好，请问您的训练脚本的链接是？

**Comment by changingivan** [comment]
> 您好，请问您的训练脚本的链接是？

https://github.com/alibaba/Pai-Megatron-Patch/blob/main/examples/qwen1_5/run_pretrain_megatron_qwen.sh

**Comment by jerryli1981** [comment]
试试在这个训练脚本中加上 --attention-dropout 0.0 和--hidden-dropout 0.0

**Comment by changingivan** [comment]
> 试试在这个训练脚本中加上 --attention-dropout 0.0 和--hidden-dropout 0.0

it works.  when add  --attention-dropout 0.0  and --hidden-dropout 0.0, the first-step loss become normal.  

Thanks Great!  @jerryli1981
